pyaedt/hfss3dlayout.py

```diff
@@ -1254,8 +1257,9 @@ def import_gds(self, gds_path, aedb_path=None, control_file=None, set_as_active=
         aedb_path : str, optional
             Full path to the AEDB file.
         control_file : str, optional
-            Path to the XML file with the stackup information. The default is ``None``, in
+            Path to the XML or tech file with the stackup information. The default is ``None``, in
```

Comment:
```suggestion
            Path to the XML or TECH file with the stackup information. The default is ``None``, in
```
Question: when is "Full path to...." used in a description versus "Path to...."? We should be consistent, and I assume that there is a difference.